apps/web/src/features/charte/ChartPage.module.css

```diff
@@ -0,0 +1,63 @@
+.container {
+  padding: 48px 0 64px;
+  background-color: var(--background-alt-blue-france);
+  text-align: center;
+
+  @media (min-width: 768px) {
+    padding: 64px 0 80px;
+  }
+
+  h1 {
+    color: var(--blue-france-sun-113-625);
+  }
+}
+
+.tileContainer {
+  background: var(--light-background-default-grey, #fff);
+  padding: 48px;
+  border-radius: 16px;
+  text-align: left;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.chartCardsContainer {
+  @media (min-width: 768px) {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 24px;
+  }
+  @media (max-width: 1024px) {
```

Comment:
that's fine for this PR but by convention : 

- we declare first the style for mobile UI without any media query 
- and then override the styles for desktop with the @media query

```css
.chartCardsContainer {
 display: flex;
    flex-direction: column;
    gap: 24px;
  @media (min-width: 768px) {
       display: grid;
       grid-template-columns: repeat(2, 1fr);
  }
  }

Reply:
done, thx !